docs/terms/terms.md

```diff
@@ -0,0 +1 @@
+# MapColonies Available Terms & Abbreviations
```

Comment:
You can insert the pages table of content here.

Reply:
done